package.json

```diff
@@ -6,21 +6,24 @@
   "scripts": {
     "start": " cd views && npm start",
     "build": "cd views && npm run build",
-    "test": "cd views && npm run test "
+    "test": "cd views && npm run test ",
+    "lint": "./node_modules/.bin/eslint app.js; exit 0"
```

Comment:
* No need for `./node_modules/.bin/eslint`. `eslint` is enough
* Replace `app.js` with `.`
* Remove `; exit 0`

Reply:
I need exit 0, to ignore errors

Reply:
Thank you for the explanation. No need to remove `; exit 0` then.

Reply:
no problem